### PR TITLE
8 create database

### PR DESCRIPTION
**Pull request by kaidobit**
- deploy postgres in docker
- add liquibase to backend
- create default, dev and docker spring profiles
- create first to tables using liquibase
- create optional database fixtures
- dockerise backend in dockerfile for compose
- documentation

### CODE REVIEW

File: packages/backend/src/main/resources/changesets/changeset-1.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        - createTable:
+            tableName: room
+            columns:
+              - column:
+                  name: id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
```
Comment (ironpinguin) [inline packages/backend/src/main/resources/changesets/changeset-1.yml:15]: We should use modern ids and not this classic auto increment integers.

Like ULID...

File: packages/backend/src/main/resources/changesets/changeset-1.yml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        - createTable:
+            tableName: session
+            columns:
+              - column:
+                  name: id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
```
Comment (ironpinguin) [inline packages/backend/src/main/resources/changesets/changeset-1.yml:28]: We should use ULID

**Review by ironpinguin — COMMENTED** [review]
This is only the migration? 
We create a database with tables and content. But without any data objects in the kotlin code?

**Comment by kaidobit** [comment]
Issue is saying >create database<

Surely I can create the entities in the code, it wasn't mentioned in the ticket

File: packages/backend/src/main/resources/changesets/fixtures-1.1.sql [hunk 3]
```diff
@@ -0,0 +1,53 @@
+INSERT INTO public.room (id, name)
```
Comment (ironpinguin) [inline packages/backend/src/main/resources/changesets/fixtures-1.1.sql:1]: Please add context to the fixtures..sql.
To ensure that if somebody change the changelog config that this fixture will be correct used.
```suggestion
--changeset fixtures:1.1 contextFilter:fixtures
INSERT INTO public.room (id, name)
```

**Review by ironpinguin — APPROVED** [review]

**Pull request merged**